app/src/oculusvr/res/layout/webxr_interstitial_controller.xml

```diff
@@ -28,12 +28,12 @@
             android:layout_height="130dp"
             >
             <ImageView
+                android:id="@+id/left_controller"
```

Comment:
This indeed should work but as I mentioned in the other comment I think it's better to do it differently.

Here you should be able to define a different RelativeLayout for the Meta Quest Pro (we select which one is shown with the condition in `app:visibleGone`. I'd basically copy everything and just modify the `app:visibleGone` and the `android:src` values for the case of the Meta Quest Pro.

Reply:
Done